src/program.js

```diff
@@ -210,6 +210,13 @@ Example: $0 --help run.
           demand: true,
           type: 'string',
         },
+        'api-proxy': {
+          describe: 
```

Comment:
I think this is the line with some white space at the end of it. Try deleting everything after `describe:`.